packages/mui-codemod/src/v7.0.0/lab-removed-components/lab-removed-components.js

```diff
@@ -0,0 +1,96 @@
+const REMOVED_EXPORTS = [
+  'Alert',
+  'AlertTitle',
+  'Autocomplete',
+  'AvatarGroup',
+  'Pagination',
+  'PaginationItem',
+  'Rating',
+  'Skeleton',
+  'SpeedDialAction',
+  'SpeedDialIcon',
+  'SpeedDial',
+  'ToggleButton',
+  'ToggleButtonGroup',
+  'usePagination',
+];
+
+function replaceComponentFileImports(j, root, exportName) {
+  root
+    .find(j.ImportDeclaration)
+    .filter((path) => path.node.source.value === `@mui/lab/${exportName}`)
+    .forEach((path) => {
+      j(path).replaceWith(
+        j.importDeclaration(path.node.specifiers, j.literal(`@mui/material/${exportName}`)),
+      );
+    });
+}
+
+function findOrCreateMaterialBarrelImport(j, root) {
```

Comment:
```suggestion
function findOrCreateMaterialNamedImport(j, root) {
```

and so on in the rest of the file.